dcos_launch/terraform.py

```diff
@@ -0,0 +1,100 @@
+from dcos_launch import util
+import yaml
+import os
+import json
+import shutil
+import uuid
+
+# TODO pinning terraform-dcos is not enough because its tf_dcos_core dependency is not pinned. Create a branch on
+# terraform-dcos that pins tf_dcos_core and replace TERRAFORM_DCOS_VERSION here by that branch name. Regularly update
+# that branch with master.
+TERRAFORM_DCOS_VERSION = 'master'
+TERRAFORM_DCOS_EE_VERSION = 'service_credentials_gcp2'
+
+
+class TerraformLauncher(util.AbstractLauncher):
+    def __init__(self, config: dict, env=None):
+        self.config = config
+        self.init_dir = config.get('init_dir')
+        if not self.init_dir:
+            self.init_dir = 'terraform-init-' + uuid.uuid4().__str__()
```

Comment:
Also seems like something we should take care of in config module as a default setter

Reply:
Done